### DISCUSSION TITLE
Nous D1T Temperature wrong

### DISCUSSION THREAD

**Discussion by jheisel01** [arendst/Tasmota#22710] [Q&A] [answered]
My **Nous D1T 16A** displays incorrect temperature values with Tasmota. It displays -15.1 °C instead of ~ 23.5 °C.
CMD: adcparam1
RSL: RESULT = {“AdcParam1”:[34,32000,10000,3350,0]}

GPIO viewer shows values between 2430 and 2440 at 24.5°C housing temperature. PIN 34 set as ADC Temp 1

Tasmota Program Version | 14.4.1 (release-tasmota32)
ESP Chip Id | 3929208 (ESP32-U4WDH-D v3.1)

Does anyone here possibly have the same model and can post its adcparam1 parameters?

All other functions work without any problems.

Regards,
Jürgen Heisel

**Comment by SBorg2014** [comment, marked as answer]
Same Device, same issue. Unfortunately i updated to a newer Tasmota-Version before i saved/checked the value. Meanwhile i used now `adcparam 1, 32000, 10000, 15000` and it seems to work. Since the sensor is internal, it also measures the heating caused by the ESP32 and not just the ambient temperature.
